merge.go

```diff
@@ -23,7 +24,55 @@ const (
 //
 // Merge operation maybe a very time-consuming operation when the database is large.
 // So it is recommended to perform this operation when the database is idle.
-func (db *DB) Merge() error {
+// If reopenAfterDone is true, the original file will be replaced by the merge file,
+// and db's index will be rebuilt after the merge is complete.
+func (db *DB) Merge(reopenAfterDone bool) error {
+	if err := db.merge(); err != nil {
```

Comment:
Two functions with the same name, `innerMerge` or `doMerge`, etc.

Reply:
ok